### PR TITLE
Add USE_PRODUCTION_API build flag to Siri Extension

### PR DESCRIPTION
**Pull request by khoi**
- Update distribution provisioning profiles for Siri Extensions
- Add USE_PRODUCTION_API flag

### CODE REVIEW

File: Toggl.Daneel.SiriExtension.UI/Toggl.Daneel.SiriExtension.UI.csproj [hunk 1]
```diff
@@ -36,13 +36,14 @@
     <DefineConstants></DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Distribution: Toggl OU (B227VTMZ94)</CodesignKey>
```
Comment (Odrakir) [inline Toggl.Daneel.SiriExtension.UI/Toggl.Daneel.SiriExtension.UI.csproj]: Why this?

File: Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj [hunk 2]
```diff
@@ -39,30 +39,30 @@
     </DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Distribution: Toggl OU (B227VTMZ94)</CodesignKey>
```
Comment (Odrakir) [inline Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj]: Why?

File: Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj [hunk 3]
```diff
@@ -39,30 +39,30 @@
...
-    <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
     <OutputPath>bin\iPhoneSimulator\Release</OutputPath>
-    <DefineConstants>
-    </DefineConstants>
+    <DefineConstants>__IOS__;__MOBILE__;__UNIFIED__;USE_PRODUCTION_API</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Distribution: Toggl OU (B227VTMZ94)</CodesignKey>
```
Comment (Odrakir) [inline Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj]: ?

File: Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj [hunk 4]
```diff
@@ -39,30 +39,30 @@
...
     </MtouchVerbosity>
+    <CodesignProvision>Toggl Mobile Siri Extension</CodesignProvision>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|iPhoneSimulator' ">
-    <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
     <OutputPath>bin\iPhoneSimulator\Release</OutputPath>
-    <DefineConstants>
-    </DefineConstants>
+    <DefineConstants>__IOS__;__MOBILE__;__UNIFIED__;USE_PRODUCTION_API</DefineConstants>
```
Comment (Odrakir) [inline Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj]: Only for `Release|iPhoneSimulator`? what about `Release|iPhone`? Shouldn't there be a `Release|AnyCPU`?

File: Toggl.Daneel.SiriExtension.UI/Toggl.Daneel.SiriExtension.UI.csproj (hunk 1 above)
Comment (khoi) [reply Toggl.Daneel.SiriExtension.UI/Toggl.Daneel.SiriExtension.UI.csproj]: that's the signing certificate for it, this is set only for `Release` configuration. `Daneel` is also doing this

File: Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj (hunk 4 above)
Comment (khoi) [reply Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj]: This is not for `Release|iPhoneSimulator` but for `Release`. There is only 2 build config for SiriExtensions, 1 is `Debug`, and 1 is `Release`.

Comment (khoi) [reply Toggl.Daneel.SiriExtension/Toggl.Daneel.SiriExtension.csproj]: nvm you're right, updated

File: Toggl.Daneel.SiriExtension.UI/Toggl.Daneel.SiriExtension.UI.csproj (hunk 1 above)
Comment (Odrakir) [reply Toggl.Daneel.SiriExtension.UI/Toggl.Daneel.SiriExtension.UI.csproj]: 👍

**Pull request merged**